tests/queuestat_input/assert_show_output.py

```diff
@@ -0,0 +1,1638 @@
+Ethernet0    UC3              52               70           19            76          N/A
+Ethernet0    UC4              11               59           12            94          N/A
+Ethernet0    UC5              36               62           35            40          N/A
+Ethernet0    UC6              49               91            2            88          N/A
+Ethernet0    UC7              33               17           94            74          N/A
+Ethernet0    UC8              40               71           95            33          N/A
+Ethernet0    UC9              54                8           93            78          N/A
+Ethernet0   MC10              83               96           74             9          N/A
+Ethernet0   MC11              15               60           61            31          N/A
+Ethernet0   MC12              45               52           82            94          N/A
+Ethernet0   MC13              55               88           89            52          N/A
+Ethernet0   MC14              14               70           95            79          N/A
+Ethernet0   MC15              68               60           66            81          N/A
+Ethernet0   MC16              63                4           48            76          N/A
+Ethernet0   MC17              41               73           77            74          N/A
+Ethernet0   MC18              60               21           56            54          N/A
+Ethernet0   MC19              57               31           12            39          N/A
+Ethernet0  ALL20             N/A              N/A          N/A           N/A          N/A
+Ethernet0  ALL21             N/A              N/A          N/A           N/A          N/A
+Ethernet0  ALL22             N/A              N/A          N/A           N/A          N/A
+Ethernet0  ALL23             N/A              N/A          N/A           N/A          N/A
+Ethernet0  ALL24             N/A              N/A          N/A           N/A          N/A
+Ethernet0  ALL25             N/A              N/A          N/A           N/A          N/A
+Ethernet0  ALL26             N/A              N/A          N/A           N/A          N/A
+Ethernet0  ALL27             N/A              N/A          N/A           N/A          N/A
+Ethernet0  ALL28             N/A              N/A          N/A           N/A          N/A
+Ethernet0  ALL29             N/A              N/A          N/A           N/A          N/A
+
+For namespace :
+     Port    TxQ    Counter/pkts    Counter/bytes    Drop/pkts    Drop/bytes    Trim/pkts
+---------  -----  --------------  ---------------  -----------  ------------  -----------
+Ethernet4    UC0              41               96           70            98            0
+Ethernet4    UC1              18               49           63            36          100
+Ethernet4    UC2              99               90            3            15          N/A
```

Comment:
N/A is for queues without trim action in the queue profile?

Reply:
@kperumalbfn `N/A` means no counter available - either counter polling is disabled or not supported by platform. This is a standard SONiC behavior